src/Microsoft.AspNet.Mvc.ModelBinding/Validation/ICompositeClientModelValidatorProvider.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNet.Mvc.ModelBinding.Validation
+{
+    /// <summary>
+    /// Aggregate of <see cref="IClientModelValidatorProvider"/>s that delegates to its underlying providers.
+    /// </summary>
+    public interface ICompositeClientModelValidatorProvider : IClientModelValidatorProvider
```

Comment:
Not used anywhere, its only there for completeness. ( there is a `ICompositeModelValidatorProvider`). 
I am fine removing it if people are not ok.